docs/src/modules/javascript/examples/action/src/action.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Action, replies } from "@lightbend/akkaserverless-javascript-sdk";
+import * as proto from "../lib/generated/proto";
+
+type Request = proto.akkaserverless.tck.model.action.Request;
+type ProcessGroup = proto.akkaserverless.tck.model.action.ProcessGroup;
+
+const { Response, ProcessGroup } = proto.akkaserverless.tck.model.action;
+
+export const tckModel: Action = new Action(
```

Comment:
The action example is using the TCK? This doesn't look that good as an example.

Reply:
Yes, it doesn't correspond to the `proto` which is shown above on the page.
I was thinking about what to do with this one. 
First I wanted to skip it entirely for TS, because adding a TS version based on the `proto` would have look strange when switching between the tabs. 

But then I've just decided to add the typed version of the TCK example to keep it consistent.